src/ui/windows/TogglDesktop/TogglDesktop/ui/views/TimerEntryListView.xaml

```diff
@@ -4,20 +4,93 @@
+                                                BorderThickness="1,1,1,1">
+                                    </Border>
+                                </StackPanel>
+                                <ControlTemplate.Triggers>
+                                    <Trigger Property="IsSelected" Value="True">
+                                        <Setter TargetName="Border" Property="BorderThickness" Value="1" />
+                                    </Trigger>
+                                    <Trigger Property="IsSelected" Value="False">
+                                        <Setter TargetName="Border" Property="BorderThickness" Value="0" />
+                                    </Trigger>
+                                </ControlTemplate.Triggers>
+                            </ControlTemplate>
+                        </Setter.Value>
+                    </Setter>
+                </Style>
+                <Style TargetType="TabControl">
+                        <Setter Property="Template">
+                            <Setter.Value>
+                                <ControlTemplate TargetType="{x:Type TabControl}">
+                                    <Grid KeyboardNavigation.TabNavigation="Local">
+                                        <Grid.RowDefinitions>
+                                            <RowDefinition Height="Auto" />
+                                            <RowDefinition Height="*" />
+                                        </Grid.RowDefinitions>
+
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition Width="*" />
+                                            <ColumnDefinition Width="Auto" />
+                                        </Grid.ColumnDefinitions>
+
+                                        <Border BorderThickness="0" Margin="0,0,0,0">
+                                            <UniformGrid Rows="1" x:Name="HeaderPanel" IsItemsHost="True"
+                                                Visibility="{Binding Path=IsTimelineViewEnabled, Converter={StaticResource BooleanToVisibilityConverter}}"/>
+                                        </Border>
+
+                                        <Border x:Name="Border"
+                                                Grid.Row="1" Grid.ColumnSpan="2"
+                                                BorderThickness="{TemplateBinding BorderThickness}"
+                                                Background="{TemplateBinding Background}"
+                                                BorderBrush="{TemplateBinding BorderBrush}">
+                                            <ContentPresenter x:Name="PART_SelectedContentHost" ContentSource="SelectedContent" />
+                                        </Border>
+                                </Grid>
+                            </ControlTemplate>
+                            </Setter.Value>

```

Comment:
The formatting (tabs) is off which makes it hard to read.

Do you think we should start using https://github.com/Xavalon/XamlStyler ?